src/index.js

```diff
@@ -32,6 +34,7 @@ const EmojiPicker = ({
   groupNames = {},
   pickerStyle = {},
   groupVisibility = {},
+  placeholder = 'Search',
```

Comment:
Nice, but I would actually default this to `null`, but allow consumers to set it. The reason is that if we introduce "search" as the default, it would change for all users, even if they don't have it at the moment.

Also, `placeholder` is kind of general, could be mean other things as well, such as the placeholder for emojis. To reduce ambiguity, maybe it is best to call it "searchPlaceholder".

```suggestion
  searchPlaceholder = null,
```